Data_exploration_template.ipynb

```diff
@@ -89,9 +98,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "DATASTORE_URL = \"http://cardshark.cs.berkeley.edu\"\n",
+    "DATASTORE_URL = \"bin/data\"\n",
     "AUTHOR_EMAIL = \"shankari@eecs.berkeley.edu\"\n",
-    "sd = eisd.SpecDetails(DATASTORE_URL, AUTHOR_EMAIL, \"train_bus_ebike_mtv_ucb\")"
+    "sd = eisd.FileSpecDetails(DATASTORE_URL, AUTHOR_EMAIL, \"train_bus_ebike_mtv_ucb\")"
```

Comment:
Can you make sure you make these changes to the other notebooks as well?

Reply:
I would also add a cell with a commented out call from `ServerSpecDetails` and some documentation indicating that it is an alternate mechanism.